### ISSUE TITLE
Remove global karma option in autotest-js.sh

### ISSUE THREAD

**Issue by Henni** [owncloud/core#25299]
### Steps to reproduce
1. install karma-cli globally on your system (`npm install -g karma-cli`)
2. execute `./autotest-js.sh`

### Expected behaviour
autotest-js.sh runs just fine

### Actual behaviour
autotest-js.sh crashes after installation of node_modules:
```
Error: Cannot find module 'build/node_modules/karma/lib/cli'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at requireCliAndRun (/usr/local/lib/node_modules/karma-cli/bin/karma:41:20)
    at /usr/local/lib/node_modules/karma-cli/bin/karma:78:14
    at /usr/local/lib/node_modules/karma-cli/node_modules/resolve/lib/async.js:46:14
    at process (/usr/local/lib/node_modules/karma-cli/node_modules/resolve/lib/async.js:173:43)
    at ondir (/usr/local/lib/node_modules/karma-cli/node_modules/resolve/lib/async.js:188:17)
    at load (/usr/local/lib/node_modules/karma-cli/node_modules/resolve/lib/async.js:69:43)
```

------

karma is installed locally with autotest-js.sh anyway, so this instance should always be used.
So let's simply skip `KARMA="$(which karma 2>/dev/null)"`

**Comment by DeepDiver1975** [comment]
@PVince81 can you comment on this please? THX

**Comment by PVince81** [comment]
@Henni we need a solution that also works for people not familiar with installing modules globally

**Comment by Henni** [comment]
@PVince81 I'm aware of this. Let me rephrase this:

autotest-js.sh ([line 23](https://github.com/owncloud/core/blob/master/autotest-js.sh#L23)) installs the node modules locally, *including* karma ([package.json line 14](https://github.com/owncloud/core/blob/master/build/package.json#L14)).
Because of this we never have to use the global installation and can always use the local instance:
This means we should never `KARMA="$(which karma 2>/dev/null)"` ([line 25](https://github.com/owncloud/core/blob/master/autotest-js.sh#L25))
Instead we should always `KARMA="$PREFIX/node_modules/karma/bin/karma"` ([line 30](https://github.com/owncloud/core/blob/master/autotest-js.sh#L30))

**Comment by PVince81** [comment]
Ah yes, that makes sense. Mind doing a PR ?
